utils/measure/measure.py

```diff
@@ -299,7 +395,38 @@ def should_resume(self, csv_file_path: str) -> bool:
 
 
     def get_resume_variation(self, csv_file_path: str) -> Variation:
-        """Determine the variation where we have to resume the measurements"""
+        """This method returns the variation to resume at.
+
+        It reads the last row from the CSV file and converts it into a Variation object.
+
+        Parameters
+        ----------
+        csv_file_path : str
+            The path to the CSV file
+
+        Returns
+        -------
+        Variation:
+            The variation to resume at. None if no resuming is needed.
+
+        Raises
+        -------
+        FileNotFoundError, Exception, ZeroDivisionError, ValueError, TypeError, IndexError
+
+        Examples
```

Comment:
Examples can be removed as this is a method used internally, no user is going to call this by himself.
It should be self explanatory for each programmer how to use this method without any additional example.

Reply:
Examples are for developers, not for end-users.

Rationale is enhance the explanation and enhance reusability/testing.

That [example I gave in the main discussion](https://github.com/numpy/numpy/blob/5028e407bf7c98662bfe247895292087e1372737/numpy/matrixlib/defmatrix.py#L1036) has also examples. They are also only meant for programmers who may wanna reuse the function or check the functionality - not for end users creating Jupyter books :)